test/test.cc

```diff
@@ -322,6 +322,8 @@ std::vector< testsweeper::routines_t > routines = {
     { "scale_row_col",      test_scale_row_col, Section::aux },
     { "",                   nullptr,           Section::newline },
 
+    { "matgen",             test_matgen,       Section::aux_gen },
```

Comment:
It probably makes more sense to move `matgen` to the end, after all the `set` routines. Also, add a line:
```
    { "",                   nullptr,           Section::newline },
```
Check what `./tester -h` prints out. Currently, `matgen` is on a line with `set`.